### PR TITLE
Create Vrt.be.xml

### PR DESCRIPTION
**Pull request by jordidg**

### CODE REVIEW

**Comment by J0WI** [comment]
@jordidg please have a look at https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#enumerating-subdomains

**Comment by J0WI** [comment]
You may split `deredactie.be` and `sporza.be` into separate ruleset files.

File: src/chrome/content/rules/Deredactie.be.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="Deredactie.be">
+    <target host="deredactie.be" />
+    <target host="m.deredactie.be" />
```
Comment (J0WI) [inline src/chrome/content/rules/Deredactie.be.xml]: Please also add `hetvooruitzicht.deredactie.be` and a comment for non-working `www.deredactie.be`

File: src/chrome/content/rules/Sporza.be.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="Sporza.be">
+    <target host="sporza.be" />
+    <target host="m.sporza.be" />
```
Comment (J0WI) [inline src/chrome/content/rules/Sporza.be.xml]: See also:
```
live.sporza.be
m.live.sporza.be
m.video.sporza.be
www.sporza.be

File: src/chrome/content/rules/Vrt.be.xml [hunk 3]
```diff
@@ -1,16 +1,10 @@
-<ruleset name="VRT">
+<ruleset name="VRT.be">
```
Comment (J0WI) [inline src/chrome/content/rules/Vrt.be.xml]: See also:
```
vrt.be
www.vrt.be
aha.vrt.be
bedrijfsbezoek.vrt.be
bigband.vrt.be
communicatie.vrt.be
communicatieradio.vrt.be
nws.vrt.be
pers.vrt.be
radiospotcel.vrt.be
sandbox.vrt.be
syndicatie.vrt.be
vrtpop.vrt.be

**Comment by J0WI** [comment]
See aslo:
```
vrtnieuws.net (mismatch)
www.vrtnieuws.net (mismatch)
media.vrtnieuws.net

File: src/chrome/content/rules/Vrt.be.xml (hunk 3 above)
Comment (jordidg) [inline src/chrome/content/rules/Vrt.be.xml]: Most of these already redirect to HTTPS, should there be rules for these as well? Or a comment?

Comment (J0WI) [inline src/chrome/content/rules/Vrt.be.xml]: Please add them anyway since they are not (yet) supporting preloaded HSTS.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**